test/pprintast/raw_identifiers/test.ml

```diff
@@ -0,0 +1,42 @@
+open Ppxlib
+
+let identifier = Longident.Lident "mod"
+;;
+[%%ignore]
+
+Format.asprintf "%a" Pprintast.longident identifier
+;;
+[%%expect{|
+- : string = "\\#mod"
+|}]
+
+module Build = Ast_builder.Make(struct let loc = Location.none end)
+
+(* 10 mod 3 *)
+let expr =
+  let open Build in
+  eapply (pexp_ident (Located.mk identifier)) [(eint 10); (eint 3)]
+;;
+[%%ignore]
+
+Format.asprintf "%a" Pprintast.expression expr
+;;
+[%%expect{|
+- : string = "10 mod 3"
+|}]
+
+(* [let f = (mod) *)
```

Comment:
I would propose to also test `(mod)` (or `lsl` for a change)  in pattern position `let f (lsl) = (mod)`.